toolchain/lower/function_context.h

```diff
@@ -88,10 +88,14 @@ class FunctionContext {
   }
 
   // Returns a lowered type for the given type_id.
+  // TODO: Use GetTypeOfInst where possible.
```

Comment:
Not sure I follow this comment - the two functions seem non-overlapping (this one maps `SemIR::TypeId` to `llvm::Type*`, and `GetTypeOfInst` maps `SemIR::InstId` to `SemIR::TypeId`) - so how would one replace a use of `GetType` with `GetTypeOfInst`? (or is this comment suggesting that the argument to `GetType` should come from `GetTypeOfInst`?)